playbooks/ansible.cfg

```diff
@@ -10,4 +10,6 @@ roles_path=../../ansible-roles/roles:../../ansible-private/roles:../../ansible-r
 ansible_managed=This file is created and updated by ansible, edit at your peril
 
 [ssh_connection]
-ssh_args=-o ControlMaster=auto -o ControlPersist=60s -o ControlPath="~/.ansible/tmp/ansible-ssh-%h-%p-%r" -o ServerAliveInterval=30
+pipelining = True
+
+ssh_args=-C -o ControlMaster=auto -o ControlPersist=60s -o ControlPath="~/.ansible/tmp/ansible-ssh-%h-%p-%r" -o ServerAliveInterval=30
```

Comment:
~@thraxil What's the `-C` for.~

It's for [compression](https://explainshell.com/explain?cmd=ssh+-C). Nice!